state/state_store.go

```diff
@@ -6,6 +6,7 @@
 package state
 
 // StateStore is an interface to perform operations on store
+// nolint:golint
```

Comment:
I would love to rename this struct to just `State`, but need to refactor in all implementation for redis, cosmodb, etc. So just leave it for now.

Reply:
Agree. let's keep it as is because it requires dapr change as well.

Reply:
This can be changed and will not affect Dapr until its updated to use the new version of contrib.
However, I would advise against using the name `State` for this interface, as I don't think its descriptive enough in regards to the methods that make up the interface.

For me, `State` might represent the actual state and is mutable. A `StateStore` is more representative of the entity that gets, saves, deletes, etc. the state itself.

Reply:
oh actually, what I meant is to change it to `Store`, previous comment is typo.

Reply:
@yaron2 @youngbupark I assume you guys are happy to make it `Store` :)

Reply:
@yaron2 

> This can be changed and will not affect Dapr until its updated to use the new version of contrib.

It is true, but someone who want to develop with the component-contrib could fail because of this renaming.. :) 

@yaron2 @sayboras Yeah I overlooked @sayboras' comment on my phone. I thought it was `store` because package name is `state`. 

For renaming stuff, I would use the dedicated PR which is only for renaming because if somethings goes wrong, it is easier to revert the commit...

Reply:
you are right, I leave it as it is now, will make the changes in subsequent PR.

Reply:
> you are right, I leave it as it is now, will make the changes in subsequent PR.

@sayboras Please open the issue first to track to rename it.

Reply:
Here it is https://github.com/dapr/components-contrib/issues/67, I will start working on this after this PR is merged.